com/yadro/HWManager/Fan.interface.yaml

```diff
@@ -52,3 +52,10 @@ properties:
         - readonly
       description: >
           Upper speed limit for Fan PWM, 30..100%
+    - name: presencePinName
```

Comment:
I dislike this approach. This doesn't need to made public or semi-public. This information needs to be contained with either this package itself, or stored in the board configuration in entity-manager.

However, I'll accept this  for now, and we'll think about refactoring later in a dedicated task that I have just created.